.github/workflows/license-lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: license-lint
+on: [push, pull_request]
```

Comment:
any ignore paths we can add here like markdowns etc?

Reply:
yes, I think we can take the reverse of this. 

only run if `go.mod`/ `go.sum` and/ or `vendor` are modified.

Reply:
I added these paths; this new wf won't actually run on this now since those paths are not touched. I recommend local testing w act or I will stand up a test branch

Reply:
Gets triggered as expected:

- https://github.com/acpana/gatekeeper/tree/acpana/license-lint-2-test-branch
- https://github.com/acpana/gatekeeper/actions/runs/3857393395/jobs/6574736347